### PR TITLE
feat(contact identification): added to interaction event the identification type toggle

### PR DESCRIPTION
**Pull request by MayaSasson**
In addition changed the contact model's fields names

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ManualContactsForm/ContactForm/ContactForm.tsx [hunk 1]
```diff
@@ -1,43 +1,44 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ManualContactsForm/ContactForm/ContactForm.tsx:1]: import order

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ManualContactsForm/ContactForm/ContactForm.tsx [hunk 2]
```diff
@@ -1,43 +1,44 @@
...
 import { FormControl, Grid, MenuItem, Select } from '@material-ui/core';
 
+import Toggle from 'commons/Toggle/Toggle';
 import ContactType from 'models/ContactType';
 import StoreStateType from 'redux/storeStateType';
 import FormInput from 'commons/FormInput/FormInput';
 import InteractedContact from 'models/InteractedContact';
-import useContactFields from 'Utils/vendor/useContactFields';
+import useContactFields from 'Utils/Contacts/useContactFields';
+import IdentificationTypes from 'models/enums/IdentificationTypes';
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ManualContactsForm/ContactForm/ContactForm.tsx:12]: import order

**Review by ItayBenMoshe — DISMISSED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventSchema.ts [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import * as yup from 'yup';
 
+import IdentificationTypes from 'models/enums/IdentificationTypes';
 import placeTypesCodesHierarchy from 'Utils/placeTypesCodesHierarchy';
-import { isIdValid } from 'Utils/auxiliaryFunctions/auxiliaryFunctions';
-import InteractionEventDialogFields from 'models/enums/InteractionsEventDialogContext/InteractionEventDialogFields';
+import { isIdValid , isPassportValid, idLength, maxIdentificationLength, idBasicValidation } from 'Utils/auxiliaryFunctions/auxiliaryFunctions';
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventSchema.ts:5]: import order

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventSchema.ts [hunk 4]
```diff
@@ -59,11 +60,24 @@ const interactionEventSchema = yup.object().shape({
...
+                .max(maxIdentificationLength, `דרכון מכיל ${maxIdentificationLength} ספרות בלבד`)
+                .test('isValid', 'דרכון לא תקין', (id) => isPassportValid(id)) ,
+              otherwise: 
+                yup
+                .string()
+                .nullable()
+                .matches(idBasicValidation, 'ת.ז חייבת להכיל ספרות בלבד')
+                .length(idLength, `ת.ז מכילה ${idLength} ספרות בלבד`)
+                .test('isValid', 'ת.ז לא תקינה', (id) => isIdValid(id)),
+            }),
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionEventSchema.ts:79]: maybe export to a common schema, seems to be in multiple places

File: server/src/DBService/ContactEvent/Query.ts [hunk 5]
```diff
@@ -106,8 +106,8 @@ query getEventAndPeopleByInvestigationID($currInvestigation: Int!) {
           personByPersonInfo {
             firstName
             identificationNumber
+            identificationType
             lastName
-            gender
```
Comment (RoeiRom) [inline server/src/DBService/ContactEvent/Query.ts:110]: why removing gender

**Review by RoeiRom — CHANGES_REQUESTED** [review]

**Review by RoeiRom — APPROVED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

**Pull request merged**